markets/storageadapter/ondealsectorcommitted.go

```diff
@@ -37,6 +49,10 @@ func OnDealSectorPreCommitted(ctx context.Context, api getCurrentDealInfoAPI, ev
 			return true, false, nil
 		}
 
+		// Save the proposed deal start epoch so we can timeout if the deal
+		// hasn't been activated by that epoch
+		proposedDealStartEpoch = deal.Proposal.StartEpoch
```

Comment:
note: you don't need to do this in checkFun since deal proposal is already a parameter to this function